src/authorizations/components/redesigned/ResourceAccordion.tsx

```diff
@@ -72,32 +86,40 @@ class ResourceAccordion extends Component<Props, State> {
             permissions={permissions[resource]}
             onToggleAll={this.handleToggleAll}
           />
-          <GetResources resources={[ResourceType[resourceName]]}>
-            {!isEmpty(permissions[resource].sublevelPermissions) &&
-              this.getAccordionBody(resourceName, resource)}
-          </GetResources>
+          {resourceName === 'Telegrafs' || resourceName === 'Buckets' ? (
+            <GetResources resources={[ResourceType[resourceName]]}>
+              {!isEmpty(permissions[resource].sublevelPermissions) &&
+                this.getAccordionBody(resourceName, resource)}
+            </GetResources>
+          ) : null}
```

Comment:
not a huge deal, but i think it's more conventional in our codebase to do: 

```suggestion
          {resourceName === 'Telegrafs' || resourceName === 'Buckets' && (
            <GetResources resources={[ResourceType[resourceName]]}>
              {!isEmpty(permissions[resource].sublevelPermissions) &&
                this.getAccordionBody(resourceName, resource)}
            </GetResources>
          )}
```